core/scripts/local/RevealVote.js

```diff
@@ -0,0 +1,54 @@
+/**
```

Comment:
+1 another useful script

Side note: we've not got a bunch of these scripts you've created for doing atomic action on our protocol. Should we re-consider the directory structure for these? the `local` directory is getting a bit crowded and has a bunch of unrelated things in it.

Reply:
Yes we should group them into `/emp` and `/dvm` for example but that's outside the scope of this PR imo. I'll make an issue